### ISSUE TITLE
unable to validate against any pod security policy: [spec.securityContext.fsGroup: Invalid value: 

### ISSUE THREAD

**Issue by davidkarlsen** [hazelcast/charts#27]
I get 
```
Events:
  Type     Reason        Age                     From                   Message
  ----     ------        ----                    ----                   -------
  Warning  FailedCreate  2m14s (x17 over 4m57s)  replicaset-controller  Error creating: pods "hazelcast-mancenter-5d5bcfd7d7-" is forbidden: unable to validate against any pod security policy: [spec.securityContext.fsGroup: Invalid value: []int64{100100}: group 100100 must be in the ranges: [{1 65535}]]
```

when attempting to install the mancenter

**Comment by leszko** [comment]
What is your k8 environment? GKE? Minikube? In any case, we'll probably just change the default user to `65535`, but I'm asking where it's worth to test it.

**Comment by davidkarlsen** [comment]
I'm running on ICP, IBM Cloud Private (onprem).
https://www.ibm.com/support/knowledgecenter/en/SSBS6K_3.1.2/

**Comment by davidkarlsen** [comment]
Works fine if I override the UIDs

**Comment by leszko** [comment]
Ok, I've sent the fix #28
